en/reference/introduction.rst

```diff
@@ -158,7 +158,7 @@ Building the model
 ------------------
 
 Models are plain PHP classes. Note that you have several ways to define the mapping.
-For easy readability, we use the annotation mapping in this tutorial::
+For easy readability, we use the annotation mapping with PHPCR namespace in this tutorial::
```

Comment:
Could you fix the double colon at the end?

Reply:
the double colon is intentional. in re-structured text its the shortcut to tell that a php code segment follows.